zbus_macros/src/proxy.rs

```diff
@@ -717,12 +717,13 @@ fn gen_proxy_property(
         let receive_method = match emits_changed_signal {
             PropertyEmitsChangedSignal::True | PropertyEmitsChangedSignal::Invalidates => {
                 let (_, ty_generics, where_clause) = m.sig.generics.split_for_impl();
-                let receive = format_ident!("receive_{}_changed", method_name);
+                let receive = format_ident!("receive_{method_name}_changed");
```

Comment:
please either remove unrelated changes or split them into their own commits. As our contributing guidelines say, we prefer atomic commits (each logical change goes into its own commit).

Reply:
Reverted in f1a09ad7bd6e5465d1e0ffdc9d2926ece7637f3d.